subsys/bluetooth/controller/ll_sw/ull_slave.c

```diff
@@ -335,6 +337,13 @@ void ull_slave_done(struct node_rx_event_done *done, u32_t *ticks_drift_plus,
 					       (EVENT_JITTER_US << 1) +
 					       preamble_to_addr_us);
 	}
+
+	if (conn->tx_head ||
+	    memq_peek(conn->lll.memq_tx.head, conn->lll.memq_tx.tail, NULL)) {
+		return 0;
+	}
+
+	return conn->lll.latency;
 }
```

Comment:
There really seems to be no advantage in packing the latency decision inside this function, as it doesn't rely on the input parameters or "extract timing from done". Moving the 3-4 lines to ull_conn.c would then contain the change in one file.

Reply:
Agreed. I did have the same thought, but decided, being a slave feature to keep it in ull_slave. I will move it into ull_conn.c.